app/routes/pages/components/Sidebar.tsx

```diff
@@ -16,49 +12,24 @@ type Props = {
   handleClose: () => void;
 };
 
-class Sidebar extends Component<Props, State> {
-  render() {
-    const {
-      categorySelected,
-      currentUrl,
-      pageHierarchy,
-      isOpen,
-      handleClose,
-    }: Props = this.props;
-    // const pictureLabel = 'Hemmelig bilde';
-    return (
-      <div
-        className={isOpen ? styles.sidebarWrapper : undefined}
-        onClick={handleClose}
-      >
-        <div
-          className={cx(styles.side, isOpen && styles.isOpen)}
-          onClick={(event) => {
-            // Just werkz
-            event.stopPropagation();
-          }}
-        >
-          <aside className={styles.sidebar}>
-            <div className={styles.sidebarTop}>
-              <button className={styles.sidebarCloseBtn} onClick={handleClose}>
-                <Icon name="close" size={50} />
-              </button>
-              <h2 className={styles.sidebarHeader}>Om Abakus</h2>
-            </div>
+const Sidebar = (props: Props) => {
+  const { categorySelected, currentUrl, pageHierarchy, isOpen, handleClose } =
+    props;
```

Comment:
You can destruct these props directly and then implicitly return the JSX below. Doesn't really matter, it changes nothing and readability isn't really improved either, but still nice to know. 

```suggestion
const Sidebar = ({ categorySelected, currentUrl, pageHierarchy, isOpen, handleClose }: Props) => {

```